Parse the complete hostname when set from a file (busybox-1.10.2 behaviour)
By Dennis Groenen <tj.groenen@gmail.com> - 2011-08-19
---

--- a/networking/hostname.c
+++ b/networking/hostname.c
@@ -35,7 +35,7 @@ static void do_sethostname(char *s, int
 //		return;
 	if (isfile) {
 		parser_t *parser = config_open2(s, xfopen_for_read);
-		while (config_read(parser, &s, 1, 1, "# \t", PARSE_NORMAL & ~PARSE_GREEDY)) {
+		while (config_read(parser, &s, 1, 1, "# \t", PARSE_NORMAL)) {
 			do_sethostname(s, 0);
 		}
 		if (ENABLE_FEATURE_CLEAN_UP)
